src/codegen/compile.ml

```diff
@@ -7519,11 +7583,11 @@ module MakeSerialization (Strm : Stream) = struct
             *)
             get_thing ^^ set_result ^^
             get_memo ^^ get_result ^^ store_unskewed_ptr ^^
-            get_memo ^^ compile_add_const 4l ^^ Blob.lit env (typ_hash t) ^^ store_unskewed_ptr
+            get_memo ^^ compile_add_const 4l ^^ Blob.lit env Tagged.T (typ_hash t) ^^ store_unskewed_ptr
```

Comment:
Alternatively, we could also consider the hash fingerprint (to double check the aliasing deserialization mechanism) as pure blob?

Reply:
Sure.